alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -409,16 +409,19 @@ protected function getRenderer($flavors)
 	
 	protected function filterStreamIdsByBitrate ($streams)
 	{
-		$streamIds = array();
 		if (!$streams || !count($streams))
 		{
 			return;
 		}
-		elseif (!$this->getDynamicAttributes()->getMaxBitrate() && !$this->getDynamicAttributes()->getMinBitrate())
+		
+		$this->sanitizeRequestedFlavorParamsIds($streams);
```

Comment:
Why not do this call after you already have the filtered out the streamIds.
Also currently, the min|max bitrate logic overrides the flavorParamsIds logic (i.e in case both are provided in the playManifest URL).